src/instrumenter/custom_maps_templates.ts

```diff
@@ -698,6 +730,27 @@ function makeSetFun(
     const mkKeysLen = () => factory.makeMemberAccess("<missing>", mkKeys(), "length", -1);
     const mkKeysPush = () => factory.makeMemberAccess("<missing>", mkKeys(), "push", -1);
     const mkKeyIdxM = () => mkStructFieldAcc(factory, factory.makeIdentifierFor(m), struct, 2);
+    const mkSum = () => mkStructFieldAcc(factory, factory.makeIdentifierFor(m), struct, 3);
+
+    if (valueT instanceof IntType) {
+        // TODO: There is risk of overflow/underflow here
```

Comment:
Question: is there a still `TODO` or it is handled by current `if` branch?

Reply:
Yep there is still risk. For now we are accepting this as a feature, and communicating it to the user by renaming the function to `unchecked_sum`